### DISCUSSION TITLE
Single Quotes Spacing

### DISCUSSION THREAD

**Discussion by Tamnac** [Lisnoti/Lisnoti#1] [General]
Just an issue I noticed, single quotes have no space around them, meaning for if you have code like `'j'` or `'7'`, the two appear merged.

Here's an example 
![image](https://github.com/Lisnoti/Lisnoti/assets/49466795/aa3190f3-386a-468d-bbde-e4abba5659f8)

**Comment by TimGord** [comment]
Thank you for pointing this out.

Because I cannot reproduce this issue in any of the software I use on my Windows machine, would it be possible to let me know the following:

1. The OS used, i.e. Windows, Apple macOS or Linux.

1. The software used.

1. Does the same rendering issue occur in any other software you use?

If you were feeling generous with your time, it would really help me to know if the same issue occurs with [Noto Sans](https://fonts.google.com/noto/specimen/Noto+Sans) (on which Lisnoti is based). That way I would know whether I've created the problem or inherited it from Noto Sans.

Cheers

**Reply by Tamnac** [reply]
It was in vscode, but can no longer replicate my own issue. Maybe it was just bugging because I had just installed it. Feel free to ignore the issue now.

**Reply by TimGord** [reply]
That's helpful. Thanks for checking.

Cheers
